docs/phylum_analyze.md

```diff
@@ -0,0 +1,42 @@
+---
+title: phylum analyze
+category: 6255e67693d5200013b1fa3e
+hidden: false
+---
+
+Submit a request for analysis to the processing system
```

Comment:
Nit: In some files, this line ends with a period, but in other files it does not.

Reply:
Thank you, I think I have sorted all of these now.